collector.go

```diff
@@ -123,6 +138,10 @@ func (cc *ChunkedCollector) Flush() error {
 		if err := cc.Collector.Collect(spanID, p...); err != nil {
```

Comment:
From what I can judge in the [implementation](https://sourcegraph.com/github.com/sourcegraph/appdash@b66bf440f5f10774f883be87adb4b9ce6d3e3771/.tree/collector.go?seldef=%2Fgithub.com%2Fsourcegraph%2Fappdash%40b66bf440f5f10774f883be87adb4b9ce6d3e3771%2F.GoPackage%2Fgithub.com%2Fsourcegraph%2Fappdash%2F.def%2FChunkedCollector%2FCollect) the runtime of `Collect` is approximetly bounded by [MinInterval](https://sourcegraph.com/github.com/sourcegraph/appdash@b66bf440f5f10774f883be87adb4b9ce6d3e3771/.GoPackage/github.com/sourcegraph/appdash/.def/ChunkedCollector/MinInterval)